reliability-v2/tasks/Tasks.py

```diff
@@ -59,17 +86,17 @@ def new_project(self,user,namespace):
         kubeconfig_admin = global_data.kubeconfigs["kubeadmin"]
         kubeconfig = global_data.kubeconfigs[user]
         self.logger.info(f"[Task] User {user}: new project '{namespace}'")
-        retry = 5
-        while retry > 0:
-            result,rc = oc(f"get project {namespace} --no-headers",kubeconfig_admin,ignore_log=True,ignore_slack=True)
-            if "Terminating" in result:
-                sleep(20)
-                retry -= 1
-            else:
-                # issue during test: in about 1/10 cases, last step gets no project, but
-                # next step shows already exists. Sleep 20s can mitigate but not avoid this issue.
-                sleep(20)
-                break
+        # retry = 5
```

Comment:
@qiliRedHat Starting here, are these all commented out and left here for a reason?

Reply:
Those are the old logics , in this pr I added the verify_project_deletion function to replace this part. I leave them commented here in case I missed something.